contentcuration/contentcuration/frontend/settings/pages/Storage/index.vue

```diff
@@ -160,5 +160,8 @@
       height: 8px !important;
     }
   }
-
-</style>
+  .kexternal-redirect{
+    margin-left: -8px;
+    display: inline !important;
```

Comment:
The` !important` property might not be necessary if there are no conflicting styles. It's generally best to avoid !important unless absolutely necessary. In fact, if you use the !important  property , it will override all previous styling rules for that specific property on that element :)

Reply:
And `display` property was indeed unnecessary, so I've removed it.